std/algorithm/comparison.d

```diff
@@ -706,15 +709,19 @@ Compares two ranges for equality, as defined by predicate $(D pred)
 */
 template equal(alias pred = "a == b")
 {
+    import std.typecons: allSatisfy;
```

Comment:
Note the space typecons and allSatisfy:

    import std.typecons: allSatisfy
vs
    import std.typecons : allSatisfy

But there is no Phobos convention for this so I'd say leave it as-is.